tests/integration/suite/daprd/tracing/binding/output.go

```diff
@@ -42,6 +42,8 @@ type output struct {
 	daprd   *daprd.Daprd
 
 	traceparent atomic.Bool
+	baggage     atomic.Bool
```

Comment:
Please do not pollute existing tests, and create new ones.

Reply:
I do not think adding baggage support here is polluting the existing test. I added baggage checks to the existing output binding test because baggage and traceparent are part of the same tracing context and follow the same propagation path. Rather than duplicating the entire test setup and structure, I extended the existing test to verify both aspects of trace propagation. This maintains the test's focus on verifying trace context propagation thru the output binding while adding coverage for baggage.

Reply:
I'm going to maintain that we have different trace and baggage tests to improve test readability, clarity and maintainability.